src/server/transaction.cc

```diff
@@ -1194,6 +1200,32 @@ bool Transaction::NotifySuspended(TxId committed_txid, ShardId sid) {
   return false;
 }
 
+void Transaction::LogJournalOnShard(EngineShard* shard) {
+  // TODO: For now, we ignore non shard coordination.
+  if (shard == nullptr)
```

Comment:
There is no flow now in the code in which shard = null right?

Reply:
Yes